event_sink_clickhouse/signals.py

```diff
@@ -2,11 +2,18 @@
 Signal handler functions, mapped to specific signals in apps.py.
 """
 from django.db.models.signals import post_save
-from django.dispatch import receiver
+from django.dispatch import Signal, receiver
 
 from event_sink_clickhouse.sinks.external_id_sink import ExternalIDSInk
+from event_sink_clickhouse.sinks.user_retire import UserRetirementSink
 from event_sink_clickhouse.utils import get_model
 
+try:
+    from openedx.core.djangoapps.user_api.accounts.signals import USER_RETIREMENT_LMS_MISC
+except ImportError:
+    # Tests don't have the platform installed
+    USER_RETIREMENT_LMS_MISC = Signal()
+
```

Comment:
We should try to use an openedx event for this one, I think there isn't one but I'm not sure. I can do the contribution if needed